src/tailscale/install.sh

```diff
@@ -5,7 +5,15 @@
 
 set -euo pipefail
 
-tailscale_url="https://pkgs.tailscale.com/stable/tailscale_${VERSION}_amd64.tgz"
+platform=$(uname -m)
+if [ "$platform" = "x86_64" ]; then
+    tailscale_url="https://pkgs.tailscale.com/stable/tailscale_1.66.3_amd64.tgz"
+elif [ "$platform" = "aarch64" ] || [ "$platform" = "arm64" ]; then
+    tailscale_url="https://pkgs.tailscale.com/stable/tailscale_1.66.3_arm64.tgz"
+else
+    echo "Unsupported platform: $platform"
```

Comment:
why does this hardcode 1.66.3 where it had ${VERSION} instead before?

Reply:
ah good catch, i forgot about the version env var 😅